lib/cfg-grammar.y

```diff
@@ -1079,6 +1085,33 @@ string_or_number
         | LL_FLOAT                              { $$ = strdup(lexer->token_text->str); }
         ;
 
+existing_path
+	: string
+	  {
+            struct stat buffer;
+            int ret = stat($1, &buffer);
+            CHECK_ERROR((ret == 0), @1, "File \"%s\" does not exist", $1);
```

Comment:
I would prefer writing the textual representation of errno here.

Reply:
@furiel : Yes, that will help in debugging.